guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,252 @@
+
+{% include tip/open.html color="blue" icon="info" label="The wallet configuration is privacy-sensitive" %}
+
+A third party with access to the configuration can import the wallet and monitor all activity, past and future. However, without the necessary private keys, they cannot spend funds.
+
+{% include tip/close.html %} 
+
+## Recovery kit
+Our application goes a bit further than only providing the users with a wallet descriptor. It generates a recovery kit that contains all necessary information in a variety of formats, so that users can easily restore a wallet in case of need. The recovery kit is a ZIP folder that includes:
+
+- **Wallet backup files** compatible with third-party applications like Sparrow, Electrum or Nunchuk
+- **A recovery PDF** containing: 
+    - Wallet information and descriptor (as a QR code and in text form)
+    - Details about the two key sets and individual signing keys
+    - Visual representation of the wallet's unlocking rules
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-pdf.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-pdf@2x.png"
+   modalImage = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-pdf@2x.png"
+   alt-text = "An example of the recovery PDF file."
+   width = 1600
+   height = 800
+   modalWidth = 4050
+   modalHeight = 2168
+%}
+
+If anyone needs to recover the wallet, they can use the recovery PDF to quickly understand the wallet structure and import it into their preferred wallet software.
+
+## Backup strategy
+
+Alice and Bob want to ensure that they can restore the wallet themselves, if they ever need to. They also want to make sure that they have a robust setup in place for their children to access their inheritance.
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-scenarios.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-scenarios@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-scenarios-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-scenarios-mobile@2x.png"
+   alt-text = "A diagram showing the sequence of recovery scenarios."
+   width = 1304
+   height = 480
+%}
+
+To achieve this goal, their backup strategy accounts for the following three scenarios: 
+
+1. **Alice and Bob need to recover the wallet themselves**: They might simply need to restore the wallet because one (or both) of them lost access to their phone or computer. This route also accounts for one of the most most likely scenarios in which one spouse outlives the other 
```

Comment:
```suggestion
1. **Alice and Bob need to recover the wallet themselves**: They might simply need to restore the wallet because one (or both) of them lost access to their phone or computer. This route also accounts for one of the most likely scenarios in which one spouse outlives the other 
```